mypy/test/teststubtest.py

```diff
@@ -778,27 +792,18 @@ class Y: ...
             """,
             error="Y.f",
         )
-        yield Case(
-            stub="""
-            from typing import Tuple
-            A = Tuple[int, str]
-            """,
-            runtime="A = (int, str)",
-            error="A",
-        )
+        yield Case(stub="A = Tuple[int, str]", runtime="A = (int, str)", error="A")
```

Comment:
IMO easier to read with each kw on its own line

Reply:
This is the way black formats it, unless I make it a triple-quoted string just for its own sake. But personally I find it more readable if it fits into a single line; I guess it's pretty subjective

Reply:
Yeah, black [collapses it unless you add a trailing comma](https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#the-magic-trailing-comma). But as you said, it's subjective, if single line is easier for you then 🚀 .

Reply:
I believe this setting that we have enabled would mean that black would strip away the trailing comma even if I tried to add it in :)

https://github.com/python/mypy/blob/c517b86b9ba7487e7758f187cf31478e7aeaad47/pyproject.toml#L23